### ISSUE TITLE
Using Epona early (perhaps without bow) removes your sword from B

### ISSUE THREAD

**Issue by Schattenmensch** [az64/mm-rando#15]
Setting String: 1qgzyq-67c-79-
Starting Item is Eponas Song in this seed, but using her early on empties your B button since you don't have the bow while riding her ... and it stays empty after you get off of her. Sword is obviously still in the inventory, but I can't seem to find a way to get it back on B.
Can't test if this is definitely the missing bow since the bloody thing is in Ikana Grotto in this seed, but i highly suspect it is ... you probably either need to add the logic that the bow is required for the song and that the song can't be the starting item ... or stop her from eating my sword ^^

**Comment by ahhhreptar** [comment]
Resetting with song of time doesn't fix this issue, either.

**Comment by sivle15** [comment]
Yeah its because you rode her without the bow. That happens on vanilla majoras mask if you use glitches to get eponas song early as well, and you ride her without getting the bow first.
